README.md

```diff
@@ -54,6 +54,10 @@ installed):
     % ./utils/gen-test-certs.sh
     % ./runtest --tls
 
+To build and test crc 64 performance improvements:
+
+    % make distclean && make SEVER_TEST=yes
```

Comment:
Typo?

```suggestion
    % make distclean && make SERVER_TEST=yes
```

Btw, if we document `src/valkey-server test` for crc64, we should probably document it for all of the tests that run in this way.

Maybe it's enough to document it a comment in the Makefile? The README only documents what can be useful for end users. It doesn't include running with sanitizers and a lot of other things.

Reply:
I will update the docs.

Reply:
For reference, I'm trying to move away from this strategy: https://github.com/valkey-io/valkey/pull/344. I would actually prefer just leaving them undocumented for now.